src/com/theironyard/novauc/Eagle.java

```diff
@@ -0,0 +1,12 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by craigslistbuyer on 2/7/17.
+ */
+public class Eagle{
+    String name;
+    int age;
+    boolean isfly;
+    int length;
+    boolean isbrownandyellow;
+}
```

Comment:
Great job Andre.  the format for naming variables is lowercase to start the first word and uppercase to start all the following words.  so line 12 should be boolean isBrownAndYellow.